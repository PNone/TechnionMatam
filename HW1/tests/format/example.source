baraa regev 30 20:40
regev adi 15 20:30
daniel baraa 100 20:20
baraa daniel 20 20:20