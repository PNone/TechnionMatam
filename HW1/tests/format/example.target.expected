BlockChain Info:
1.
Sender Name: baraa
Receiver Name: regev
Transaction Value: 30
Transaction timestamp: 20:40
2.
Sender Name: regev
Receiver Name: adi
Transaction Value: 15
Transaction timestamp: 20:30
3.
Sender Name: daniel
Receiver Name: baraa
Transaction Value: 100
Transaction timestamp: 20:20
4.
Sender Name: baraa
Receiver Name: daniel
Transaction Value: 20
Transaction timestamp: 20:20
